pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -707,6 +729,8 @@ message Image {
     repeated string repo_digests = 3;
     // The size of the image in bytes.
     optional uint64 size = 4;
+    // The uid that will run the command(s).
+    optional int64 uid = 5;
```

Comment:
Actually, reading more it looks like this is strictly only UID and k8s just doesn't support `non_root_user` for images that have a non-numeric user (hey, another Dockerfile feature that's not fully supported :).

looks like my concern isn't valid.